host-bmc/host_pdr_handler.cpp

```diff
@@ -1838,6 +1877,92 @@ void HostPDRHandler::setOperationStatus()
     }
 }
 
+std::shared_ptr<EntityAuxiliaryNames>
+    HostPDRHandler::parseEntityAuxNamesPDR(std::vector<uint8_t>& pdrData)
```

Comment:
this is code duplication. I think its better to use the existing code by moving the parse function into the utils.cpp

Reply:
Yes this API almost matches with https://github.com/openbmc/pldm/blob/master/platform-mc/terminus.cpp#L307 except that in our use case we are not trimming the name. As of now I am moving this to a util API.